node_examples/infinite_stream.js

```diff
@@ -0,0 +1,221 @@
+const fs = require('fs');
+const revai = require('revai-node-sdk');
+const { Writable } = require('stream');
+
+// CUSTOMER CONSTANTS, SET THIS
```

Comment:
```suggestion
// USER CONSTANTS, SET THIS
```
I'm also not a fan of the caps lock, so maybe
```suggestion
// Set access token here.
```